elide-async/src/main/java/com/yahoo/elide/async/operation/AsyncQueryOperation.java

```diff
@@ -76,10 +75,9 @@ public void nullResponseCheck(ElideResponse response) {
     /**
      * Execute the Async Query Request.
      * @param queryObj AsyncAPI type object.
-     * @param user User object.
-     * @param apiVersion Api Version.
+     * @param scope RequestScope.
      * @return response ElideResponse object.
      * @throws URISyntaxException URISyntaxException Exception.
      */
-    public abstract ElideResponse execute(AsyncAPI queryObj, User user, String apiVersion)  throws URISyntaxException;
+    public abstract ElideResponse execute(AsyncAPI queryObj, RequestScope scope)  throws URISyntaxException;
```

Comment:
Did this contract change so that we can extract the requestHeaders and use when making calls to elide.get/QueryRunner.run.